src/daria/corrections/shape/curvature.py

```diff
@@ -380,7 +380,7 @@ def compute_stretch(self, img: Optional[np.ndarray] = None, **kwargs):
         pt_src = kwargs.pop("point_source", [Ny, Nx])
         pt_dst = kwargs.pop("point_destination", [Ny, Nx])
         stretch_center = kwargs.pop("stretch_center", [round(Ny / 2), round(Nx / 2)])
-
+        # TODO ADD WARNING!!
```

Comment:
for next time, I would not provide such commits.

Reply:
I agree. It will be fixed today.